assets/model_monitoring/components/tests/e2e/test_model_data_collector_preprocessor_e2e.py

```diff
@@ -67,7 +67,7 @@ class TestMDCPreprocessorE2E:
         "input_data, start_time, end_time",
         [
             # traditional model
-            (DATA_ASSET_IRIS_MODEL_INPUTS_WITH_DRIFT, "2023-01-29T00:00:00Z", "2023-02-03T00:00:00Z"),
```

Comment:
is this expected?

Reply:
oh I did git add . it adds all files, will remove